tests/phpunit.xml

```diff
@@ -1,21 +1,20 @@
-<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" encoding="utf-8"?>
 <phpunit bootstrap="./bootstrap.php"
-         verbose="true"
-         colors="true"
-         timeoutForSmallTests="900"
-         timeoutForMediumTests="900"
-         timeoutForLargeTests="900">
-	<testsuite name='Password Policy Tests'>
-		<directory suffix='Test.php'>.</directory>
-	</testsuite>
-	<!-- filters for code coverage -->
-	<filter>
-		<whitelist>
-			<directory suffix=".php">../lib</directory>
-		</whitelist>
-	</filter>
-	<logging>
-		<!-- and this is where your report will be written -->
-		<log type="coverage-clover" target="./clover.unit.xml"/>
-	</logging>
+  verbose="true"
+  colors="true"
+  timeoutForSmallTests="900"
+  timeoutForMediumTests="900"
+  timeoutForLargeTests="900">
+  <coverage>
+    <include>
+      <directory suffix=".php">../lib</directory>
+    </include>
+    <report>
+      <!-- and this is where your report will be written -->
+      <clover outputFile="./clover.unit.xml"/>
+    </report>
+  </coverage>
+  <testsuite name="Password Policy Tests">
+    <directory suffix="Test.php">.</directory>
+  </testsuite>
```

Comment:
Nothing changed but indentation? :see_no_evil:

Reply:
There are some structure-changes...
Basically `phpunit --migrate-configuration` did this.

Reply:
Don't like it, we use tabs everywhere ;)

Reply:
Well indentation is no problem to change... 👍